### PR TITLE
jupyter notebook convertr for .ipynb files to pdf,html,python,markdow…

### LINKED ISSUE #1093: jupyter notebook convertor
# Description

Making a Jupyter notebook converter to convert jupyter notebooks (.ipynb files) to pdf, HTML, python, markdown.

| name            | problem/motivation                                                  | possible solution  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Jupyter notebook converter | Some time in practicals we need the output to send to our teacher for submission we can't directly send jupyter file so using my program convert it into pdf, python, HTML, markdown and send it(i faced this challenge for submissions)  | I have created python program which uses some python libraries and converts Jupyter notebook into pdf, HTML, python, markdown | @venkateshpensalwar |

**Screenshots/Proof of Concept**

convert1

### PR DESCRIPTION
**Pull request by venkateshpensalwar**
# Description

jupyter notebook converter to convert .ipynb files to pdf, markdown, HTML, python files.

Fixes #1093 

## Type of change

- [ ✔️ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ✔️ ] My code follows the style guidelines(Clean Code) of this project
- [✔️  ] I have performed a self-review of my own code
- [✔️  ] I have commented my code, particularly in hard-to-understand areas
- [✔️  ] I have created a helpful and easy to understand `README.md`
- [✔️  ] My changes generate no new warnings
- [✔️  ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by venkateshpensalwar** [comment]
@vybhav72954  can you tell me how i can remove errors from this PR checking i am new to this thing

**Comment by iamakkkhil** [comment]
create a function for your file conversion and use main function to drive the code

**Comment by iamakkkhil** [comment]
create a requirements.txt file

File: Python/Jupyter_Notebook_Convertor/notebook_convertor.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""This Program helps user to convert jupyter notebook to HTML,MarkDown,PDF,python file"""
```
Comment (vybhav72954) [inline Python/Jupyter_Notebook_Convertor/notebook_convertor.py:1]: We don't need a script docstring.

**Comment by venkateshpensalwar** [comment]
Ok I will remove

**Comment by venkateshpensalwar** [comment]
i removed @vybhav72954 Docstring

File: Python/Jupyter_Notebook_Convertor/notebook_convertor.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+
```
Comment (iamakkkhil) [inline Python/Jupyter_Notebook_Convertor/notebook_convertor.py:2]: remove these lines, and I already requested some changes

**Review by iamakkkhil — CHANGES_REQUESTED** [review]

**Comment by venkateshpensalwar** [comment]
Yes I will do my health is not well give me some days

**Comment by venkateshpensalwar** [comment]
Yes I will do my health is not well give me some days

**Comment by HarshCasper** [comment]
Closing the PR because of no updates.

**Pull request closed without merging**